### PR TITLE
DOC: Several clarifications and syntax fixes

### PR DESCRIPTION
**Pull request by sappelhoff**
While going through the source, docs, and examples I noticed several places for improvement.

### CODE REVIEW

File: examples/plot_estimate_global_reject.py [hunk 1]
```diff
@@ -39,6 +39,9 @@
 # Now we get the rejection dictionary
 
 from autoreject import get_rejection_threshold  # noqa
+
+# We could use the `decim` parameter to only take every nth time slice
+# this would speed up the computation time at the expense of accuracy
```
Comment (jasmainak) [inline examples/plot_estimate_global_reject.py]: not necessarily unless you have very peaky artifacts ...

**Comment by jasmainak** [comment]
While you're at it, would you mind updating CircleCI and Travis to be Python 3 compliant?

**Comment by jasmainak** [comment]
Otherwise changes LGTM

File: autoreject/autoreject.py [hunk 2]
```diff
@@ -806,8 +813,7 @@ class AutoReject(object):
         If None, defaults to `np.linspace(0, 1.0, 11)`
     n_interpolate : array | None
         The values to try for the number of channels for which to interpolate.
-        This is :math:`\\rho`.If None, defaults to
-        np.array([1, 4, 32])
+        This is :math:`\\rho`.If None, defaults to np.array([1, 4, 32])
```
Comment (agramfort) [inline autoreject/autoreject.py]:
```suggestion
        This is :math:`\\rho`. If None, defaults to np.array([1, 4, 32])
```

File: examples/plot_estimate_global_reject.py (hunk 1 above)
Comment (sappelhoff) [reply examples/plot_estimate_global_reject.py]: agreed, I updated my comment to reflect this.

**Comment by sappelhoff** [comment]
> While you're at it, would you mind updating CircleCI and Travis to be Python 3 compliant?

I started doing it but ran into problems. I'll do it in another PR. Also trying to fix the caching with circleci, which is not working:

image

**Comment by sappelhoff** [comment]
so this one is ready to be merged. CIs will be fixed in #145.

Errors in this PR's CI are not related to the changes.

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Pull request merged**